### ISSUE TITLE
Integrate debian packaging

### ISSUE THREAD

**Issue by rjwhelan** [panda-re/panda#1089]
We have a debian packaging process we'd like to integrate. This issue will track development.

This could be a first step to creating a PANDA PPA on Launchpad.

**Comment by AndrewFasano** [comment]
A PPA sounds like a great start. A few wish-list items (feel free to ignore):
1) Update CI to produce debs on PRs to `dev` and `stable` branches
2) Either split PyPANDA and core panda (including libpanda) into two debs or make the PyPANDA pip package depend on the core panda deb package.

**Comment by Ben-Dumas** [comment]
this is still something we are interested in integrating.

**Comment by Ben-Dumas** [comment]
The debian packaging we (MITLL) use depends on building panda and plugins separately, [this is the state of that issue](https://github.com/panda-re/panda/issues/1088). We previously used just classic debian packaging, though, which i've implemented [here](https://github.com/panda-re/panda/tree/debian-packaging). This works for building the binary and plugins, but I'm not finding a clean way to add header files to the package. Header files would be necessary if you wanted to develop plugins without rebuilding panda. It would work for pushing to a PPA, though.

**Comment by AndrewQuijano** [comment]
Seems like Debian Packages are now implemented

https://github.com/panda-re/panda/pull/1411/files

and we can now create PyPanda wheel and deploy it to PyPi

https://github.com/panda-re/panda/pull/1435/files
https://github.com/panda-re/panda/pull/1481/files

I am also updating the Debian Package to follow standard practices, e. g. more complete control file, dynamic versioning, placing files in `/usr` instead of `/usr/local`

https://github.com/panda-re/panda/pull/1564/files

I'd agree to have some header files, least it would be nice to have the Panda Hypercall Struct there for LAVA to use (and avoid code duplication). Currently, it seems the final panda container does NOT have /usr/local/include with the headers, should the install step in Make be updated to include these as well?

https://github.com/panda-re/panda/tree/dev/panda/include/panda

**Comment by lacraig2** [comment]
done

**Comment by AndrewQuijano** [comment]
Headers will be added to Debian package once #1617 is merged
